libs/wallet-ui/src/index.css

```diff
@@ -75,3 +75,13 @@ https://create-react-app.dev/docs/adding-images-fonts-and-files/
     width: 14px;
   }
 }
+
+.scrollbar-hide::-webkit-scrollbar {
+  display: none;
+}
+
+/* For IE, Edge and Firefox */
+.scrollbar-hide {
```

Comment:
**Praise (non-blocking)**: was surprised to see this in an 'app' but a few seconds thought about it (code reuse, cross platform web views) made me realise why it's here.